.gitlab-ci.yml

```diff
@@ -0,0 +1,64 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v1
+    file: '/.gitlab-ci-template.yml'
+
+variables:
+  SERVICE_PORT: 80
+  POSTGRES_ENABLED: 0
+
+build-review:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-review"
+    K8S_SECRET_REACT_APP_API_URI: "https://venepaikka-api.test.hel.ninja/graphql_v2/"
+  only:
+    refs:
+      - branches
+  except:
+    refs:
+      - develop
+      - master
+
+build-staging:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-staging"
+    K8S_SECRET_REACT_APP_API_URI: "https://venepaikka-api.test.hel.ninja/graphql_v2/"
```

Comment:
Should be `DOCKER_BUILD_ARG_REACT_APP_API_URI: ...`.

Reply:
Changed